### PR TITLE
feat: Added apply section

### PR DESCRIPTION
**Pull request by HashirGJ8842**

### CODE REVIEW

**Comment by HashirGJ8842** [comment]
Initial Form:-
image

When something is filled incorrectly:-
image

When form is submitted successfully:-
image

Admin section, for management of records:-
image

When adding a member in admin section:-
image

When deleting a member in admin section:-
image

When updating a member from admin section:-
image

**Comment by iamareebjamal** [comment]
Firebase DB security rules are missing

**Comment by HashirGJ8842** [comment]
> Firebase DB security rules are missing
@iamareebjamal 
I wrote the rules in `rules.bolt` first, now i wrote them in `database.rules.json`, did i do it right?

**Comment by iamareebjamal** [comment]
You have to generate JSON from bolt rules

**Comment by HashirGJ8842** [comment]
Oh, okay then, I'll revert the changes i made in json and write them in `rules.bolt` instead.

File: rules.bolt [hunk 1]
```diff
@@ -227,7 +227,36 @@ path /inventory {
...
-    write() { true }
+    read() {
+        isAdmin() || isCurrentUser(data.child('uid').val())
+    }
+    path /{members} is Members {
+        write() { true }
+    }
+}
+
+type Members {
```
Comment (iamareebjamal) [inline rules.bolt:241]:
```suggestion
type Member {
```

File: rules.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
+{
```
Comment (iamareebjamal) [inline rules.json:1]: Why added this file?

File: rules.bolt (hunk 1 above)
Comment (HashirGJ8842) [reply rules.bolt:241]: I didn't do that because there is one `type member` present in line 188 :sweat_smile:

**Comment by HashirGJ8842** [comment]
> You have to generate JSON from bolt rules

I'm sorry, i misunderstood you, i thought i had to generate and all the JSON file. Removing `rules.json` now.

File: rules.bolt (hunk 1 above)
Comment (iamareebjamal) [reply rules.bolt:241]: Make this ClubMember then

Comment (HashirGJ8842) [reply rules.bolt:241]: Okay :+1:

**Comment by iamareebjamal** [comment]
Great job. Nice to see the default quality of code contribution increasing :)

**Comment by HashirGJ8842** [comment]
> Great job. Nice to see the default quality of code contribution increasing :)

All thanks to all the reviews you gave :)

**Pull request merged**